### PR TITLE
update python workshop instructions

### PR DESCRIPTION
**Pull request by Tianyi-W**
Fixes # 

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: workshop/content/30-python/30-hello-cdk/200-lambda.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ Let's go to the AWS Lambda Console and test our function.
 
     ![](./lambda-2.png)
 
-4. Select __Amazon API Gateway AWS Proxy__ from the __Event template__ list.
+4. Select __apigateway-aws-proxy__ from the __Event template__ list.
```
Comment (peterwoodworth) [inline workshop/content/30-python/30-hello-cdk/200-lambda.md:201]: The template's actual name is `apigateway-aws-proxy`, but the dropdown the user will see still shows the full name `API Gateway AWS Proxy`
Screen Shot 2022-06-17 at 4 39 27 PM

I think I'd prefer to either clarify both, or keep the name as the same as the name seen in the dropdown menu

**Review by peterwoodworth — CHANGES_REQUESTED** [review]
I'm pretty concerned about the dimensions of the images here. Could you try to keep the new images around the same height and width as the previous ones?

**Comment by peterwoodworth** [comment]
@Tianyi-W Are you planning on getting back to this?

**Comment by Tianyi-W** [comment]
> 
Yes, I am planning on getting back to this.

File: workshop/content/30-python/30-hello-cdk/200-lambda.md [hunk 2]
```diff
@@ -198,7 +198,7 @@ Let's go to the AWS Lambda Console and test our function.
 
     ![](./lambda-2.png)
 
-4. Select __Amazon API Gateway AWS Proxy__ from the __Event template__ list.
+4. Select __API Gateway AWS Proxy__ from the __Event template__ list.
```
Comment (peterwoodworth) [inline workshop/content/30-python/30-hello-cdk/200-lambda.md:201]: This change would also be necessary in the other workshops - at the least it is in TypeScript. Could you check the others and update there as well?

**Review by peterwoodworth — CHANGES_REQUESTED** [review]
These images you've added are great since they are on the newer console. What was the reasoning behind changing the specific images you did? Are there not other images that need updating?

Additionally, the other language workshops use the same images for the most part. If we are going to replace the images for one language, we should do it for all langauges. Please make sure the images you're changing are updated in all languages
Lastly, there's no need to rename the old images. You can just delete them

**Comment by peterwoodworth** [comment]
Ping me if you plan to work on these changes again

**Pull request closed without merging**